pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -155,7 +155,7 @@ func NewManager(cpuPolicyName string, reconcilePeriod time.Duration, machineInfo
 		if err != nil {
 			return nil, err
 		}
-		klog.Infof("[cpumanager] detected CPU topology: %v", topo)
+		klog.InfoS("detected CPU topology", "topology", topo)
```

Comment:
Ahhh I forgot to say this explicitly and I hate to block over this but we need to capitalize each log line. Like so:

```suggestion
		klog.InfoS("Detected CPU topology", "topology", topo)
```

Some of them already are, they all need to be consistently capitalized.

Reply:
Capitalized. ✅
😄